### DISCUSSION TITLE
右手側トラックボールが認識されない

### DISCUSSION THREAD

**Discussion by hiro1115** [Yowkees/keyball#473] [Q&A]
前回質問させていただいたものです。
remapで右手側設定を行うと使用できたのですが、今日になり右手側のトラックボールの認識がされなくなりました。右手側のキー入力は可能です。
修正すべき点を教えていただけますと幸いです。よろしくお願いします。
![image](https://github.com/Yowkees/keyball/assets/142844754/e3c9ebc6-1e09-4166-9629-1ba5a8d5fe4a)

**Comment by Yowkees** [comment]
トラックボール回りの半田付けに少々不安定な部分がある感じでしょうか。
トラックボール基板とProMicro裏、ProMicro本体の半田付けを見直してみていただければ幸いです。
お手数ですがご確認お願いします。

**Reply by hiro1115** [reply]
ショートしていそうな部分と、はんだ不良のありそうなところをやり直したところ改善いたしました。誠にありがとうございました。

**Comment by Yowkees** [comment]
ご自身で工夫して修正いただきありがとうございました。無事に動くようになり安心しました。
ぜひ末永くご愛用いただければ幸いです。
